selenium/README.md

```diff
@@ -51,3 +51,106 @@ and run the `analyze` method to get results.
 - `withOnlyRules` limites rules run to those specified.
```

Comment:
https://github.com/dequelabs/axe-core-maven-html/issues/284 - will be revamping the Selenium docs